pavelib/prereqs.py

```diff
@@ -185,3 +185,4 @@ def install_prereqs():
     install_ruby_prereqs()
     install_node_prereqs()
     install_python_prereqs()
+    sh("pip uninstall -y --disable-pip-version-check south")
```

Comment:
The south could be installed due to django-splash. I'm working on fork to upgrade splash to 1.8